src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1

```diff
@@ -457,9 +457,16 @@ class ADOSVTBase: SVTBase {
 		#perform bug logging after control scans for the current resource
 		if ($BugLogParameterValue) 
 		{
-			if (($ControlResults.ControlResults.VerificationResult -contains "Failed") -or ($ControlResults.ControlResults.VerificationResult -contains "Verify")) {
-				$this.BugLoggingPostEvaluation($ControlResults, $BugLogParameterValue)
+			#added check azuretable check here, if (azuretable enabled and scan mode is ca) then only allow bug logging.
+			#If not cseo then go to bug logging. 
+			#if cseo then go to only if azuretable enabled and scan mode is ca
+			if (($this.SubscriptionContext.SubscriptionName -ne [Constants]::CSEOOrg) -or ( ($this.SubscriptionContext.SubscriptionName -eq [Constants]::CSEOOrg) -and [Helpers]::CheckMember($this.ControlSettings.BugLogging, "UseAzureStorageAccount") -and ($this.ControlSettings.BugLogging.UseAzureStorageAccount -eq $true -and ([AzSKSettings]::GetInstance().GetScanSource() -eq "CA") ) ) ) 
```

Comment:
enter only if UseAzureStorageAccount is true and scan mode is CA
or enter if UseAzureStorageAccount is false